Makefile

```diff
@@ -0,0 +1,34 @@
+.PHONY: build-garage-release run-garage-release
+
+GARAGE_RELEASE="v2018.10.0"
+
+build-garage-release: docker/Dockerfile.release docker/mjkey.txt
+	docker build \
+		-f docker/Dockerfile.release \
+		--build-arg RELEASE="${GARAGE_RELEASE}" \
+		-t rlworkgroup/garage:release .
```

Comment:
tags (garage:release) are for specifying versions, not variants. if you want an image variant, you should name it with a dash.

for instance, the ci version of this Dockerfile is garage-ci

Reply:
I followed this nomenclature in the new changes